fuse/plugins/detector_physics/csv_input.py

```diff
@@ -69,6 +69,18 @@ class ChunkCsvInput(FuseBasePlugin):
         help="n_interactions_per_chunk",
     )
 
+    first_chunk_left = straxen.URLConfig(
```

Comment:
This is just putting these parameters in lineage. In principle, they are not related to this PR.